docs/resources/service.md

```diff
@@ -61,6 +61,7 @@ resource "clickhouse_service" "service" {
 - `num_replicas` (Number) Number of replicas for the service. Available only for 'production' services. Must be between 3 and 20. Contact support to enable this feature.
 - `password` (String, Sensitive) Password for the default user. One of either `password` or `password_hash` must be specified.
 - `password_hash` (String, Sensitive) SHA256 hash of password for the default user. One of either `password` or `password_hash` must be specified.
+- `release_channel` (String) Release channel to use for this service. Either 'default' or 'fast'. Only supported on 'production' services.
```

Comment:
Let's make a note in the document about this restriction: `Switching from 'fast' to 'default' release channel is not supported`.

Reply:
makes sense, done